pkgs/development/python-modules/flask-mysqldb/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, flask
+, mysqlclient
+}:
+
+buildPythonPackage rec {
+  pname = "flask-mysqldb";
+  version = "1.0.1";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
FYI: A new parameter, pyproject, has recently been introduced.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Thanks for the hint! I use the new parameter now.